### PR TITLE
@starsirius: orchestration presenter for GET /accounts/:id v2

### LINKED ISSUE #7: Clients should be able to read an API V2 GET accounts/:id endpoint that wraps account detail and its relations
- [x] routes
- [x] features
- [x] messages

### LINKED ISSUE #9: Account updated_at should be touched when a relation is created/modified

### PR DESCRIPTION
**Pull request by dylanfareed**
This API endpoint will permit the mobile apps to get all the content for each account in a single call. 

Also, notably,

* Moves shared `belongs_to` relations for `Feature`, `Message`, and `Route` into an `Accountable` concern.
* touch accounts when relation is modified
* adds [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers) for activerecord spec convenience

Thanks!

image

Closes #9 
Closes #7

### CODE REVIEW

**Comment by dylanfareed** [comment]
:see_no_evil: at the time it takes to run the spec suite here.

image

**Comment by starsirius** [comment]
Nice!!! The next goal is to run the specs faster than rubocop!

File: app/models/concerns/accountable.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module Accountable
+  extend ActiveSupport::Concern
+  included do
+    belongs_to :account, touch: true
```
Comment (starsirius) [inline app/models/concerns/accountable.rb]: Nice `touch: true`~

**Comment by orta** [comment]
hah, nice!

File: spec/models/feature_spec.rb [hunk 2]
```diff
@@ -1,8 +1,6 @@
 require 'spec_helper'
 
 describe Feature do
-  it 'validates that an account is present' do
-    expect(subject).to be_invalid
-    expect(subject.errors.messages).to eq(account: ["can't be blank"])
-  end
+  it { should belong_to(:account).touch(true) }
```
Comment (starsirius) [inline spec/models/feature_spec.rb]: This shoulda matchers look pretty useful! :+1:

File: app/models/concerns/accountable.rb (hunk 1 above)
Comment (dylanfareed) [inline app/models/concerns/accountable.rb]: Nice and simple, yeah!

**Pull request merged**